src/app/games/game-summaries/external-game/external-game.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { IExternalGame } from '../../games.service.interface';
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
+import { faChartLine, faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { GamesService } from '../../games.service';
+
+/**
+ * Displays a game's name, decomp progress, and links.
+ */
+@Component({
+  selector: 'external-game',
+  templateUrl: './external-game.component.html',
+  styleUrls: ['./external-game.component.scss']
+})
+export class ExternalGameComponent implements OnChanges {
+
+  constructor(private gamesService: GamesService) { }
+
+  /**
+   * The game data to use.
+   */
+  @Input() data: IExternalGame;
+  /**
+   * Shield data
+   */
+  shield: string;
+  /**
+   * The percentage to display for completion.
+   */
+  total?: string;
+  /**
+   * Icon definition.
+   */
+  github = faGithub;
+  /**
+   * Icon definition.
+   */
```

Comment:
```suggestion
  /**
   * Icon definition.
   */
  github = faGithub;
```
both comments are not needed